db/migrate/20191023162335_remove_paperclip_columns.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class RemovePaperclipColumns < ActiveRecord::Migration[6.0]
+  def up
+    remove_columns :assets,
+                   :file_file_name,
```

Comment:
It seems we have an index for searching on this column, will this delete the index as well?

db/migrate/20151021085335_add_search_query_indexes.rb:
add_gist_index :assets, :file_file_name

Reply:
"Indexes and table constraints involving the column will be automatically dropped as well"